tests/conftest.py

```
@@ -53,6 +53,39 @@ def _is_xdist_worker(config: pytest.Config) -> bool:
+    if not workspace:
+        return  # only relevant in CI, where test-results/ is uploaded
+    dest = Path(workspace) / "test-results" / "worker-logs" / item.name
+    try:
+        for suffix, prefix in (("", ""), ("_frontend", "frontend_")):
+            src = Path(resolve_test_output_path(item.name + suffix))
+            if not src.is_dir():
+                continue
+            for log in src.glob("*.log*"):
+                dest.mkdir(parents=True, exist_ok=True)
+                shutil.copy2(log, dest / (prefix + log.name))
+    except Exception as exc:  # never let log copying fail the run
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)